src/js/components/sharedComponents/Pagination.jsx

```diff
@@ -146,7 +147,7 @@ export default class Pagination extends React.Component {
         if (pager.currentPage === pager.endPage) {
             rangeEnd = pager.totalItems;
         }
-        const resultsText = `${rangeStart}-${rangeEnd} of ${this.props.totalItems} results`;
+        const resultsText = `${rangeStart}-${rangeEnd} of ${formatNumberWithPrecision(this.props.totalItems, 0)} results`;
```

Comment:
Could you format the page numbers as well? 
<img width="268" alt="screen shot 2018-07-25 at 8 31 14 am" src="https://user-images.githubusercontent.com/7108785/43200947-25b26874-8fe5-11e8-99dd-366bec5ca57b.png"> eg `19,368`